Framework/PythonInterface/mantid/kernel/src/Exports/Logger.cpp

```diff
@@ -28,33 +27,33 @@ boost::shared_ptr<Logger> getLogger(const std::string &name) {
   return boost::make_shared<Logger>(name);
 }
 
-boost::shared_ptr<Logger> create(const boost::python::object &name) {
-  return boost::make_shared<Logger>(Converters::pyObjToStr(name));
+boost::shared_ptr<Logger> create(const std::string &name) {
+  return boost::make_shared<Logger>(name);
 }
 
-void fatal(Logger *self, const boost::python::object &message) {
-  self->fatal(Converters::pyObjToStr(message));
-}
+// void fatal(Logger *self, const boost::python::object &message) {
```

Comment:
Remove commented out code from this file

Reply:
Thanks!